src/main/java/com/xeeva/catalog/ItemDetailsPage.java

```diff
@@ -1,169 +1,167 @@
+	@FindBy(xpath = ".//*[@class='textFieldList']") private List<Listbox> lstUOMGlobalCart;
+	@FindBy(xpath=".//*[@class='add-to-cart-box']") private List<WebElement> btnIAddToCartItems;
+	@FindBy(xpath="//div[@id='divAppInfoMsg'][@class='addMessage']") private Label lblCartItemAddedMessage;
+	@FindBy(xpath = "//table/tbody/tr[6]/td/table/tbody/tr/td[4]/select") private Listbox lstunitofMeasure;
+	@FindBy(xpath = "//tr[1]/td[3]//tr[7]//td[4]/a/div")
+	private Button btnAddToCart;
+	@FindBy(linkText = "Global Items")	private Link globalItemsTab;
+
+	// global Items , UnitPrice ,Quantity
+	@FindBy(xpath = "//table/tbody/tr[6]/td/table/tbody/tr/td[2]/input") 
+	private Textbox txtunitPrice;
+	@FindBy(xpath = "//table/tbody/tr[7]/td/table/tbody/tr/td[4]/input") 
+	private Textbox txtQuantity;
+
+
+
+	/**Constructor**/
+	public ItemDetailsPage(OrasiDriver driver){
+		this.driver = driver;
+		ElementFactory.initElements(driver, this);
+	}
+
+	public void pageLoaded(){
+		catalogSearch.syncVisible(10, false);
+	}
+
+	/**Page Interactions**/
+
+	/**
+	 * @summary: Method to add Price Agreement Item to Cart.
+	 * @author: Praveen Namburi, @version: Created 08-09-2016
+	 */
+	public void addPriceAgrmntItemToCart(){
+		btnAddToCart.syncEnabled();
+		btnAddToCart.click();
+	}
+
+	/**
+	 * @summary Method to add non price agreement item to cart 
+	 * @author Lalitha Banda
+	 * @date 14/9/16
+	 **/
+	public void selectUOMValueAndAddNonPriceItemToCart(String strUOMValue){
+		if(lstSelectUOM.isDisplayed()){
+			lstSelectUOM.select(strUOMValue);
+			btnAddToCart.jsClick();
+			driver.setPageTimeout(3,TimeUnit.SECONDS);
+			lblCartItemAddedMessage.syncVisible(15, false);
+			String getCartItemAddedMessage = lblCartItemAddedMessage.getText();
+			System.out.println("Message after adding item to cart : "+ getCartItemAddedMessage);
+			TestReporter.assertTrue(getCartItemAddedMessage.contains("added successfully!"), "Item added to the cart.");
+		}else{
+			TestReporter.assertFalse(!lstSelectUOM.isDisplayed(),"Selected Random Item is Price aggrement Item!!");
+		}
+	}
+
+	/**
+	 * @summary: Method to Add-Item-To-Cart for Price agreement item.
+	 * @author: Praveen Namburi, @version: Created 12-09-2016
+	 */
+	public void addPriceAgreementItemToCart_And_Verify(){
+		btnAddToCart.jsClick();
+		driver.setPageTimeout(3,TimeUnit.SECONDS);
+		lblCartItemAddedMessage.syncVisible(15, false);
+		String getCartItemAddedMessage = lblCartItemAddedMessage.getText();
+		TestReporter.logStep("Message after adding item to cart : "+ getCartItemAddedMessage);
+		TestReporter.assertTrue(getCartItemAddedMessage.contains("added successfully!"), "Item added to the cart.");
+
+	}
+
+	/**
+	 * @summary Method to modify Item details for UOM,UP and quantity 
+	 * @author Lalitha Banda
+	 * @date 14/9/16
+	 **/
+
+	public void  modify_ItemDetails(String UP,String Qty,String UOM){
+		txtunitPrice.clear();
+		txtunitPrice.isDisplayed();
```

Comment:
If you are not doing anything with the return value of isDisplayed(), then there is no need to use this method.  please remove.